### PR TITLE
Licensing: Provide correct URL for AGPL license

### PR DESCRIPTION
**Pull request by witekest**
Provide correct URL for AGPL license

### CODE REVIEW

**Comment by daniellee** [comment]
@RichiH @pkolyvas just to add some context to this PR.

The licenseUrl property is used for the Open Source link in the footer:

FooterScreenshot

The license type in the footer was added years before the switch to AGPL and shows whether you are using the Open Source version of Grafana or the Enterprise version. The question is what should the link for this text be? Does it make sense to directly link to the AGPL license here?

**Comment by RichiH** [comment]
As it's not saying "license" or "AGPLv3", I would tend to link to https://grafana.com/oss/grafana/ as that's the canonical Grafana OSS landing page.

**Review by RichiH — APPROVED** [review]

**Comment by RichiH** [comment]
Thanks @witekest !

File: pkg/services/licensing/oss.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func (l *OSSLicensingService) LicenseURL(user *models.SignedInUser) string {
 		return l.Cfg.AppSubURL + "/admin/upgrading"
 	}
 
-	return "https://grafana.com/products/enterprise/?utm_source=grafana_footer"
+	return "https://grafana.com/oss/grafana"
```
Comment (marefr) [inline pkg/services/licensing/oss.go:44]: I think we should keep the utm_source query parameter for tracking purposes.

```suggestion
	return "https://grafana.com/oss/grafana?utm_source=grafana_footer"
```

**Review by marefr — APPROVED** [review]
LGTM but added a suggestion

File: pkg/services/licensing/oss.go (hunk 1 above)
Comment (marefr) [reply pkg/services/licensing/oss.go:44]: @RichiH  @daniellee @torkelo any thoughts?

Comment (torkelo) [reply pkg/services/licensing/oss.go:44]: yes, the tracking should not be removed

**Review by ying-jeanne — APPROVED** [review]

**Pull request merged**